Real Estate Webscrapper/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Requirements
+  
+1. requests==2.25.1
+2. pandas==1.2.4
+3. beautifulsoup4==4.9.3
+4. pip==20.2.3
+5. python==3.9.4
```

Comment:
this is not the way of defining the requirements file. Check out this file for ref: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt